mappings/fh.mapping

```diff
@@ -0,0 +1 @@
+CLASS fh
```

Comment:
what is this for

Reply:
maybe try the save as -> enigma (folder) option danielshe suggested in his last PR?

Reply:
meh it doesn't really matter